web.jsf12/nbproject/project.properties

```diff
@@ -15,13 +15,5 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# release.external/jsf-1.2.zip!/jsf-1.2/commons-beanutils.jar=modules/ext/jsf-1_2/commons-beanutils.jar
-# release.external/jsf-1.2.zip!/jsf-1.2/commons-collections.jar=modules/ext/jsf-1_2/commons-collections.jar
-# release.external/jsf-1.2.zip!/jsf-1.2/commons-digester.jar=modules/ext/jsf-1_2/commons-digester.jar
-# release.external/jsf-1.2.zip!/jsf-1.2/commons-logging.jar=modules/ext/jsf-1_2/commons-logging.jar
-release.external/jsf-1.2.zip!/jsf-1.2/jsf-api.jar=modules/ext/jsf-1_2/jsf-api.jar
-# release.external/jsf-1.2.zip!/jsf-1.2/jsf-impl.jar=modules/ext/jsf-1_2/jsf-impl.jar
-release.external/jsf-1.2.zip!/jsf-1.2/LICENSE-APACHE.txt=modules/ext/jsf-1_2/LICENSE-APACHE.txt
-release.external/jsf-1.2.zip!/jsf-1.2/LICENSE.TXT=modules/ext/jsf-1_2/LICENSE.TXT
-release.external/jsf-1.2.zip!/jsf-1.2/THIRDPARTYLICENSEREADME.txt=modules/ext/jsf-1_2/THIRDPARTYLICENSEREADME.txt
+release.external/jsf-api-1.2_05.jar=modules/ext/jsf-1_2/jsf-api-1.2_05.jar
```

Comment:
You are changing the filename and this in turn means, that you need to adjust the `project.xml`, as that file references the file. In this case, I would not change the filename, as the important version information is coded into the subdirectory.

The `project.xml` is also missing the `binary-origin` header (that was the case already before your changes), please also add it (see `libs.amazon/nbproject/project.xml` for an example.

Reply:
@matthiasblaesing Added jar file name in project.xml .